Trailheads/src/pages/DisplayUserInfo.page

```diff
@@ -0,0 +1,7 @@
+<apex:page id="DisplayUserInfo">
+    <apex:pageBlock>
+        <apex:pageBlockSection columns="1">
+                {! $User.FirstName}
```

Comment:
CO TO ZA SPACJA?
http://i0.kym-cdn.com/photos/images/facebook/000/787/356/d6f.jpg